src/app/helpers/object-keys-to-human-readable.helper.ts

```diff
@@ -1,13 +1,33 @@
 export function toHumanReadableKey(str: string): string {
+  // Capitalize first letter of each word
```

Comment:
Comments are unnecessary as the method name is explanatory enough and can go out of date while the code evolves. Just an observation.